.gitignore

```diff
@@ -3,6 +3,8 @@
 .DS_Store
 
 # Xcode
+*.xcodeproj/
+DerivedData/
```

Comment:
What happens if you don't add these gitignore lines, and just delete the files they're referring to?

Reply:
Appears to not add the files, I wonder why they were added before in the first place.